src/it/java/servlets/admin/config/SetCtfModeIT.java

```diff
@@ -1,272 +1,272 @@
-					{
-						String message = "CTF Mode was not actually Enabled";
-						log.fatal(message);
-						fail(message);
-					}
-					else if (ModulePlan.isOpenFloor())
-					{
-						String message = "ModulePlan still returning possitive for Open Floor Check";
-						log.fatal(message);
-						fail(message);
-					}
-				}
-				else
-				{
-					String message = "Admin was unable to enable CTF mode";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-
-	@Test
-	public void testCsrf()
-	{
-		String userName = "configAdminTester";
-		String password = userName;
-		//Verify / Create user in DB
-		try
-		{
-			TestProperties.verifyTestAdmin(log, applicationRoot, userName, password);
-			//Sign in as Admin User
-			log.debug("Signing in as User Through LoginServlet");
-			TestProperties.loginDoPost(log, request, response, userName, userName, null, lang);
-			log.debug("Login Servlet Complete, Getting CSRF Token");
-			if(response.getCookie("token") == null)
-				fail("No CSRF Token Was Returned from Login Servlet");
-			String csrfToken = response.getCookie("token").getValue();
-			if(csrfToken.isEmpty())
-			{
-				String message = new String("No CSRF token returned from Login Servlet");
-				log.fatal(message);
-				fail(message);
-			}
-			else
-			{
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost("wrongToken");
-				if(responseBody.contains("Error Occurred"))
-				{
-					log.debug("CSRF Error Occurred");
-					if(ModulePlan.isIncrementalFloor())
-					{
-						String message = "CTF Mode was enabled after what should have been a failed request";
-						log.fatal(message);
-						fail(message);
-					}
-					else if (!ModulePlan.isOpenFloor())
-					{
-						String message = "ModulePlan no longer returning possitive for Open Floor Check after what should have been a failed request";
-						log.fatal(message);
-						fail(message);
-					}
-				}
-				else
-				{
-					String message = "Registration Availability Update Failure";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-}
+package servlets.admin.config;
+
+import static org.junit.Assert.*;
+
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletConfig;
+
+import testUtils.TestProperties;
+import utils.InstallationException;
+import utils.ModulePlan;
+
+public class SetCtfModeIT
+{
+	private static org.apache.log4j.Logger log = Logger.getLogger(EnableCheats.class);
```

Comment:
Should be renamed to SetCtfModeIT too instead of EnableCheats.class (mistake in original)